src/app/components/content/IsaacCard.tsx

```diff
@@ -5,6 +5,8 @@ import { apiHelper, isAppLink } from "../../services";
 import { Link } from "react-router-dom";
 import { IsaacCardDTO } from "../../../IsaacApiTypes";
 
+// NOTE: Currently not in use by CS. To be updated
```

Comment:
I find these comments are a bit confusing. I think you're suggesting that there are plans to use this component in the future, but at the moment it's not used. If this is the case, then the comment should be updated with a reference to the ticket in which it's explained when and how this is going to be updated. And one of the items of the Acceptance Criteria in that ticket should be to remove these comments.